### PR TITLE
chore: add instructions to the README for running package tests

### PR DESCRIPTION
**Pull request by travis**
Document installing Docker Desktop and setting required environment variables.

### CODE REVIEW

**Review by vasco-santos — APPROVED** [review]

**Comment by vasco-santos** [comment]
Thanks @travis

File: README.md [hunk 1]
```diff
@@ -56,6 +56,25 @@ npm start
...
+not need to be set to real values - the following works in `bash`-like shells:
+
+```
+export AWS_REGION='us-west-2'; export AWS_ACCESS_KEY_ID='NOSUCH'; export AWS_SECRET_ACCESS_KEY='NOSUCH
+```
+
+Finally, run the tests with:
+
+```
+npm test
```
Comment (olizilla) [inline README.md:73]: From the workspace root or the package dir?

I think with `npm` the way to run per package tests is `npm test -w <path/to/package>` from the root. Doing `npm` things from the packages often doesn't work as expected.

Comment (travis) [reply README.md:73]: yep, worth documenting the per-package test command too, I'll add!

**Comment by travis** [comment]
ok made one more minor tweak, happy to update after merge if anyone has any more comments

**Pull request merged**